docs/cli/edgedb_migration.rst

```diff
@@ -70,6 +127,44 @@ The tool will find all the unapplied migrations in
 ``dbschema/migrations/`` directory and sequentially run them on the
 target instance.
 
+
+.. _ref_cli_edgedb_migrate:
+
+Show Status
+===========
+
+To figure out the status of the schema and migrations use the respective
+command:
+
+.. cli:synopsis::
+
+    edgedb -I <instance-name> show-status [migration-option...]
```

Comment:
Why does this part mention `-I <instance-name>` while previous doesn't? `edgedb migrate` below doesn't mention it too

Reply:
It's mentioned in synopsis lines, where we're trying to give comprehensive guide on the command. But in summary, it's too much noise. It's also not strictly required, you can also specify `--host=` or `--dsn=` instead.

And we are going to drop ubiquitous usage of `-I` with project-local config in a month or so, so I wouldn't concentrate on that too much.